tests/from.bats

```diff
@@ -12,12 +12,10 @@ load helpers
 
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json dir:${elsewhere})
   buildah rm $cid
-  buildah rmi ${elsewhere}
```

Comment:
Is it ok to not remove the elsewhere image?  Are we relying on the test cleanup to get rid of it?  Do we know why the buildah rmi was failing?

Reply:
I assumed that when you're using an image from a directory (`dir:`) it is not saved in `/var/run/containers/storage` and thus no need to run `buildah rmi` upon it.

I might be wrong and that's fine. Let me know and I work it out. 😃

Reply:
No I believe that it gets pulled from there and stored in containers/storage.  Could you confirm or deny my theory.  :^)

Reply:
```
# export elsewhere=/tmp/image1
# mkdir -p ${elsewhere}
# tree ${elsewhere}
/tmp/image1

0 directories, 0 files
# buildah from --name cnt1 busybox
cnt1
# buildah commit cnt1 dir:${elsewhere}
Getting image source signatures
Copying blob sha256:57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566
 706.12 KiB / 706.12 KiB [==================================================] 0s
Copying blob sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1
 32 B / 32 B [==============================================================] 0s
Copying config sha256:6209ef68c57fd2e54be91dd3393b281dcac41035ee9648d67198cc779cd0803f
 711 B / 711 B [============================================================] 0s
Writing manifest to image destination
Storing signatures
# buildah images
IMAGE ID             IMAGE NAME                                               CREATED AT             SIZE
5b0d59026729         docker.io/library/busybox:latest                         Jan 24, 2018 04:29     1.299 MB
# tree ${elsewhere}
/tmp/image1
├── 4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1.tar
├── 57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566.tar
├── 6209ef68c57fd2e54be91dd3393b281dcac41035ee9648d67198cc779cd0803f.tar
├── manifest.json
└── version

0 directories, 5 files
```

@nalind @TomSweeneyRedHat can you confirm this?